### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by brandones**

### CODE REVIEW

File: __mocks__/openmrs-esm-module-config.mock.ts [hunk 1]
```diff
@@ -1 +1,5 @@
+import React from "react";
+
 export const getDevtoolsConfig = jest.fn().mockResolvedValue({});
+
+export const ModuleNameContext = React.createContext("fake-module-config");
```
Comment (brandones) [inline __mocks__/openmrs-esm-module-config.mock.ts:5]: Substantive changes

File: jest.json [hunk 2]
```diff
@@ -1,4 +1,7 @@
 {
+  "transform": {
+    "\\.tsx?$": "babel-jest"
+  },
```
Comment (brandones) [inline jest.json:4]: Substantive changes

File: package.json [hunk 3]
```diff
@@ -1,21 +1,19 @@
...
   "main": "dist/openmrs-esm-devtools.js",
   "version": "1.2.0",
   "license": "MPL-2.0",
   "scripts": {
     "start": "webpack-dev-server",
-    "lint": "eslint . --ext tsx",
+    "lint": "eslint src --ext js,jsx,ts,tsx",
     "test": "jest --config jest.json",
     "watch-tests": "jest --config jest.json --watch",
+    "prettier": "prettier --write **/*",
```
Comment (brandones) [inline package.json:12]: Substantive changes

File: src/root.component.tsx [hunk 4]
```diff
@@ -9,6 +9,9 @@ function Root(props) {
   return <Devtools {...props} />;
 }
 
-export default openmrsRootDecorator({ featureName: "devtools" })(Root);
+export default openmrsRootDecorator({
+  featureName: "devtools",
+  moduleName: "@openmrs/esm-devtools",
+})(Root);
```
Comment (brandones) [inline src/root.component.tsx:15]: Substantive changes -- for openmrs-react-root-decorator upgrade

File: webpack.config.js [hunk 5]
```diff
@@ -35,23 +35,29 @@ module.exports = {
...
+    ],
   },
-  externals: ["react", "react-dom", /^@openmrs\/esm/],
+  externals: [
+    "react",
+    "react-dom",
+    /^@openmrs\/esm/,
+    "i18next",
+    "react-i18next",
+  ],
```
Comment (brandones) [inline webpack.config.js:51]: Substantive changes -- for openmrs-react-root-decorator upgrade

**Review by FlorianRappl — APPROVED** [review]
LGTM!

**Pull request merged**